### PR TITLE
Translate New Component doc

### PR DESCRIPTION
**Pull request by afc163**
ant-design/ant-design-pro#120

### CODE REVIEW

File: docs/new-component.en-US.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+Then you have a component `ImageWrapper` you can use at anywhere in your application.
+
+## Use it
+
+Import your component and use it like other React components.
+
+```js
+import React from 'react';
+import ImageWrapper from '../../components/ImageWrapper';  // 注意保证引用路径的正确
```
Comment (ddcat1115) [inline docs/new-component.en-US.md]: chinese

File: docs/new-component.en-US.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+export default () => (
+  <ImageWrapper
+    src="https://os.alipayobjects.com/rmsportal/mgesTPFxodmIwpi.png"
+    desc="示意图"
```
Comment (ddcat1115) [inline docs/new-component.en-US.md]: chinese

**Comment by ddcat1115** [comment]
@afc163

**Comment by afc163** [comment]
working on it

**Pull request merged**